File: Andy/src/main/java/com/menasr/org/Tost.java
package com.menasr.org;

import android.content.Context;
import android.support.annotation.NonNull;
import android.widget.Toast;

/** This is custom Toast class*/
public class Tost {
    Context context = Andy.getContext();

    /**
     * @param stringId pass String id like <b>R.string.'String name'</b>
     */
    public void toastShort(@NonNull int stringId) {
        Toast.makeText(context, Andy.RES.getString(stringId), Toast.LENGTH_SHORT).show();
    }

    /**
     * @param msg "Message" pass String id like <b>R.string.'String name'</b>
     */
    public void toastShort(@NonNull String msg) {
        Toast.makeText(context, msg, Toast.LENGTH_SHORT).show();
    }

    /**
     * @param stringId pass String id like <b>R.string.'String name'</b>
     */
    public void toastLong(@NonNull int stringId) {
        Toast.makeText(context, Andy.RES.getString(stringId), Toast.LENGTH_LONG).show();
    }

    /**
     * @param msg "Message" pass String id like <b>R.string.'String name'</b>
     */
    public void toastLong(@NonNull String msg) {
        Toast.makeText(context, msg, Toast.LENGTH_LONG).show();
    }


}

File: AndyKtx/src/main/java/com/appHelperKtx/Tost.kt
package com.appHelperKtx

import android.widget.Toast

/** This is custom Toast class */
class Tost {
    internal var context = Andy().context

    /**
     * @param stringId pass String id like **R.string.'String name'**
     */
    fun toastShort(stringId: Int) {
        Toast.makeText(context, Andy.RES.getString(stringId), Toast.LENGTH_SHORT).show()
    }

    /**
     * @param msg "Message" pass String id like **R.string.'String name'**
     */
    fun toastShort(msg: String) {
        Toast.makeText(context, msg, Toast.LENGTH_SHORT).show()
    }

    /**
     * @param stringId pass String id like **R.string.'String name'**
     */
    fun toastLong(stringId: Int) {
        Toast.makeText(context, Andy.RES.getString(stringId), Toast.LENGTH_LONG).show()
    }

    /**
     * @param msg "Message" pass String id like **R.string.'String name'**
     */
    fun toastLong(msg: String) {
        Toast.makeText(context, msg, Toast.LENGTH_LONG).show()
    }


}
